Goals.md

```diff
@@ -14,3 +14,4 @@
 - [x] Own a boat
```

Comment:
when you can make a change here!!!

Reply:
@rizel-botany what would you suggest is the best way to do this?